src/style.css

```diff
@@ -1,3 +1,110 @@
+/* Reset browser styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+}
+
+/* Header Styles */
+
+:root {
+  --white-color: #fffefe;
+  --black-color: #050505;
+  --blue-color: #00acdc;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
```

Comment:
Consider explicitly setting the height for the `.header` class to ensure consistent layout across different browsers and devices.